### PR TITLE
Add entries into static/_redirects for Bengali localization

### PR DESCRIPTION
**Pull request by Acesif**
Signed-off-by: Asif Zubayer Palak asifpalak@gmail.com
Localizes `static/_redirects`  to Bengali
This is an initial PR as may need some improvements!
Tracking issue: #31677

### CODE REVIEW

**Comment by sftim** [comment]
/retitle Add entries into static/_redirects for Bengali localization

File: content/bn/docs/tasks/tools/install-kubectl-linux.md
Comment (sftim) [inline content/bn/docs/tasks/tools/install-kubectl-linux.md:1]: This change is not related to `static/_redirects`

File: static/_redirects [hunk 1]
```diff
@@ -7,6 +7,7 @@
 /concepts/containers/container-lifecycle-hooks/     /docs/concepts/containers/container-lifecycle-hooks/ 301
 /docs/     /docs/home/ 301!
 /de/docs/     /de/docs/home/ 301!
+/bn/docs/     /bn/docs/home/ 301!
```
Comment (sftim) [inline static/_redirects:10]:
```suggestion
/bn/docs/     /bn/docs/home/ 301!
/de/docs/     /de/docs/home/ 301!
```

(In the latin alphabet, b comes before d)

**Comment by mitul3737** [comment]
Hey @sftim is this file , ok? Or, we need to add `/bn/docs/setup/learning-environment/     /bn/docs/tasks/tools/ 302!` as well?
